server/src/ingest/async-pipeline.ts

```
@@ -296,6 +297,29 @@ export class PgAsyncIngestRepository implements AsyncIngestRepository {
+    const { rows } = await this.pool.query(
+      `SELECT tenant_id, COUNT(*)::int AS count
+         FROM ingest_async_jobs
+        WHERE tenant_id = ANY($1)
+          AND status = 'PROCESSING'
+        GROUP BY tenant_id`,
+      [tenantIds],
+    );
+
+    const map = new Map<string, number>();
+    // Initialize with 0 for all requested tenants to ensure safe lookups
+    tenantIds.forEach((id) => map.set(id, 0));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)